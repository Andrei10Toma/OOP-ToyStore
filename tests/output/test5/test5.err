exceptions.CommandNotFound: Input command aplydiscount not found.
	at main.Main.interpretCommand(Main.java:85)
	at main.Main.main(Main.java:45)
exceptions.NegativePriceException: The product with price 4.5878048780487815 and id eac7efa5dbd3d667f26eb3d3ab504464 cannot have a fixed discount of 8.0 applied.
	at store.Store.applyDiscountProducts(Store.java:245)
	at command.ApplyDiscount.execute(ApplyDiscount.java:50)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.NegativePriceException: The product with price 0.7500000000000001 and id 30950ba413fbae88b190efb5c7237fc2 cannot have a fixed discount of 5.0 applied.
	at store.Store.applyDiscountProducts(Store.java:245)
	at command.ApplyDiscount.execute(ApplyDiscount.java:50)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.DiscountNotFoundException: Discount PERCENTAGE_DISCOUNT 30.0 not found.
	at command.ApplyDiscount.execute(ApplyDiscount.java:48)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.CurrencyEURCantBeChanged: The EUR currency can't be cheanged.
	at command.UpdateParity.execute(UpdateParity.java:38)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.NegativePriceException: The product with price 1.2825000000000002 and id eac7efa5dbd3d667f26eb3d3ab504464 cannot have a fixed discount of 8.0 applied.
	at store.Store.applyDiscountProducts(Store.java:245)
	at command.ApplyDiscount.execute(ApplyDiscount.java:50)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.ProductNotFound: Product with ID NuVreauRestanteSemestrulAsta not found.
	at command.CalculateTotal.execute(CalculateTotal.java:42)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.ProductNotFound: Product with id NiciSemestrulUrmator not found.
	at command.ShowProduct.execute(ShowProduct.java:41)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.ManufacturerNotFound: Manufacturer with name TomaAndrei was not found.
	at command.ListProductsByManufacturer.execute(ListProductsByManufacturer.java:53)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.DiscountNotFoundException: Discount FIXED_DISCOUNT 4.0 not found.
	at command.ApplyDiscount.execute(ApplyDiscount.java:48)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
exceptions.DuplicateCurrencyException: Currency with name USD tried to be added
	at command.AddCurrency.execute(AddCurrency.java:33)
	at command.CommandQueue.executeCommands(CommandQueue.java:25)
	at main.Main.interpretCommand(Main.java:63)
	at main.Main.main(Main.java:45)
