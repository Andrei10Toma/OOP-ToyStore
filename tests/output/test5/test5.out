USD
EUR 1.00
GBP 1.10
USD 0.82
TRY 0.11
RUB 0.01
USD
PERCENTAGE_DISCOUNT 25.0 "EVERYTHING AT 1/4" 2021-01-20T18:34:07
FIXED_DISCOUNT 8.0 "MINUS 8" 2021-01-20T18:34:07
0eb264d2d9d6651499a915942d5cd702,Cribbage board and two packs playing cards 00185,A Kent & Cleal game,$4.014,0
eeac7e98ae68235c8b1c38c0dd54e31f,Roulette & Blackjack set - 40cm. wheel, 00612,A Kent & Cleal game,$25.142,2
8e42d1931995150626eb107e78db23c0,Chess men, wood Staunton pattern, 75mm. King - 00201,A Kent & Cleal game,$5.523,0
8a6cf3ca66b9bfd4e3a5cc25356ff947,Blackjack set-00830,A Kent & Cleal game,$20.112,0
3d4688ab7b8309e13cfc1c3cfe5dbdcd,Mah jong set in walnut presentation case - 00696,A Kent & Cleal game,$90.539,0
72333b924a3cd25b23afe5020088ca7f,Mah jong set with Oriental characters-00691,A Kent & Cleal game,$45.264,2
ddbd469427906e9ffc7c7c98b502a28a,Mah jong set with bone and bamboo tiles - 00695,A Kent & Cleal game,$80.478,0
835c12f9b5d4b5e1d6a1c2ad52ae2453,Rummy set-00900,A Kent & Cleal game,$9.045,0
80f825b724cb8ce94625ed48b9aa0758,Dominoes- double six, wood,black spots, giant size-00130,A Kent & Cleal game,$13.069,0
5c6f2c519b3e37e915a45130d0d51ef9,Dominoes double nine, plastic, black spots, spinners - 00122,A Kent & Cleal game,$6.027,0
4ad0fbbcc5a6e94b298c58f16c589e6a,Dominoes- double six, plastic,black spots,spinners-00121,A Kent & Cleal game,$6.610,2
EUR 1.00
GBP 1.10
USD 0.82
TRY 0.11
RUB 0.01
0eb264d2d9d6651499a915942d5cd702,Cribbage board and two packs playing cards 00185,A Kent & Cleal game,€1.496,0
eeac7e98ae68235c8b1c38c0dd54e31f,Roulette & Blackjack set - 40cm. wheel, 00612,A Kent & Cleal game,€9.371,2
8e42d1931995150626eb107e78db23c0,Chess men, wood Staunton pattern, 75mm. King - 00201,A Kent & Cleal game,€2.059,0
8a6cf3ca66b9bfd4e3a5cc25356ff947,Blackjack set-00830,A Kent & Cleal game,€7.496,0
3d4688ab7b8309e13cfc1c3cfe5dbdcd,Mah jong set in walnut presentation case - 00696,A Kent & Cleal game,€33.746,0
72333b924a3cd25b23afe5020088ca7f,Mah jong set with Oriental characters-00691,A Kent & Cleal game,€16.871,2
ddbd469427906e9ffc7c7c98b502a28a,Mah jong set with bone and bamboo tiles - 00695,A Kent & Cleal game,€29.996,0
835c12f9b5d4b5e1d6a1c2ad52ae2453,Rummy set-00900,A Kent & Cleal game,€3.371,0
80f825b724cb8ce94625ed48b9aa0758,Dominoes- double six, wood,black spots, giant size-00130,A Kent & Cleal game,€4.871,0
5c6f2c519b3e37e915a45130d0d51ef9,Dominoes double nine, plastic, black spots, spinners - 00122,A Kent & Cleal game,€2.246,0
4ad0fbbcc5a6e94b298c58f16c589e6a,Dominoes- double six, plastic,black spots,spinners-00121,A Kent & Cleal game,€2.464,2
14a65ee16538af1a8e29032a1186e2eb,Dino Dice - Maths, Numbers & Dots (4-6 years),Dino Dice,€2.936,5
c0904c75d570f0ccc5c8b94c56889d59,Official Legend of the 5 Rings 10xD10 Dice Set (The Crab Clan),Q-Workshop,€6.368,4
5c6f2c519b3e37e915a45130d0d51ef9,Dominoes double nine, plastic, black spots, spinners - 00122,A Kent & Cleal game,€2.246,0
€11.550
PERCENTAGE_DISCOUNT 25.0 "EVERYTHING AT 1/4" 2021-01-20T18:34:07
FIXED_DISCOUNT 8.0 "MINUS 8" 2021-01-20T18:34:08
PERCENTAGE_DISCOUNT 50.0 "HALF PRICE" 2021-01-20T18:34:08
PERCENTAGE_DISCOUNT 32.0 "RANDOM SALE" 2021-01-20T18:34:08
FIXED_DISCOUNT 20.0 "ANOTHER RANDOM DISCOUNT" 2021-01-20T18:34:08
EUR
PERCENTAGE_DISCOUNT 25.0 "EVERYTHING AT 1/4" 2021-01-20T18:34:07
FIXED_DISCOUNT 8.0 "MINUS 8" 2021-01-20T18:34:08
PERCENTAGE_DISCOUNT 50.0 "HALF PRICE" 2021-01-20T18:34:08
USD
FIXED_DISCOUNT 1.0 "EVERYTHING CHEAPER" 2021-01-14T17:17:33
EUR
